wordpress-mobile/WordPress-iOS
# Adds the medium size iOS 14 Today Widget

Ref #15103 

Notes:

- The content of the widget is just a static sample
- At the moment, the distances between the labels are automatically set by the system.

To test:

- install the app on a device or simulator with iOS 14
- add a new widget, scroll down to find WordPress, tap to add and then select the medium size widget.
- make sure the added widget is consistent with the designs in #15103. See also the example below
<p align=center>
<img width=450 src=https://user-images.githubusercontent.com/34376330/96931206-8be0c680-1482-11eb-8c5d-66f939da78a5.png> 
</p>


PR submission checklist:

- [x] ~~I have considered adding unit tests where possible.~~ in a future PR.
- [x] ~~I have considered adding accessibility improvements for my changes.~~ in a future PR.
